plugin/trino-ranger/src/main/java/io/trino/plugin/ranger/RangerSystemAccessControl.java

```diff
@@ -0,0 +1,1236 @@
+                if (toExclude == null) {
+                    toExclude = new HashSet<>();
+                }
+
+                toExclude.add(column);
+            }
+        }
+
+        return toExclude == null ? columns : columns.stream().filter(((Predicate<? super String>) toExclude::contains).negate()).collect(Collectors.toSet());
+    }
+
+    /**
+     * Create view is verified on schema level
+     */
+    @Override
+    public void checkCanCreateView(SystemSecurityContext context, CatalogSchemaTableName view)
+    {
+        if (!hasPermission(createResource(view), context, RangerTrinoAccessType.CREATE, "CreateView")) {
+            AccessDeniedException.denyCreateView(view.getSchemaTableName().getTableName());
+        }
+    }
+
+    /**
+     * This is evaluated against the table name as ownership information is not available
+     */
+    @Override
+    public void checkCanDropView(SystemSecurityContext context, CatalogSchemaTableName view)
+    {
+        if (!hasPermission(createResource(view), context, RangerTrinoAccessType.DROP, "DropView")) {
+            AccessDeniedException.denyDropView(view.getSchemaTableName().getTableName());
+        }
+    }
+
+    /**
+     * This is evaluated against the table name as ownership information is not available
+     */
+    @Override
+    public void checkCanRenameView(SystemSecurityContext context, CatalogSchemaTableName view, CatalogSchemaTableName newView)
+    {
+        if (!hasPermission(createResource(view), context, RangerTrinoAccessType.ALTER, "RenameView")) {
+            AccessDeniedException.denyRenameView(view.toString(), newView.toString());
+        }
+    }
+
+    @Override
+    public void checkCanSetViewAuthorization(SystemSecurityContext context, CatalogSchemaTableName view, TrinoPrincipal principal)
+    {
+        if (!hasPermission(createResource(view), context, RangerTrinoAccessType.ALTER, "SetViewAuthorization")) {
+            AccessDeniedException.denySetViewAuthorization(view.toString(), principal);
+        }
+    }
+
+    /**
+     * This check equals the check for checkCanCreateView
+     */
+    @Override
+    public void checkCanCreateViewWithSelectFromColumns(SystemSecurityContext context, CatalogSchemaTableName table, Set<String> columns)
+    {
+        try {
+            checkCanCreateView(context, table);
+        }
+        catch (AccessDeniedException ade) {
+            AccessDeniedException.denyCreateViewWithSelect(table.getSchemaTableName().getTableName(), context.getIdentity());
+        }
+    }
+
+    @Override
+    public void checkCanSetViewComment(SystemSecurityContext context, CatalogSchemaTableName view)
+    {
+        if (!hasPermission(createResource(view), context, RangerTrinoAccessType.ALTER, "SetViewComment")) {
+            AccessDeniedException.denyCommentView(view.toString());
+        }
+    }
+
+    /**
+     *
+     * check if materialized view can be created
```

Comment:
Remove such comments that. Notice that the same message is given with `checkCanCreateMaterializedView` method name.